scenes/performance/instruments/piano_notes.gd

```diff
@@ -1,29 +1,27 @@
 extends InstrumentNotes
 
-
-signal note_spawned()
-signal note_destroyed()
-
+signal note_spawned
+signal note_destroyed
```

Comment:
Keep initial extra newline at line 3. Same as before, do we want to remove these parentheses?